packages/vite/src/node/server/openBrowser.ts

```diff
@@ -59,45 +56,46 @@ function executeNodeScript(scriptPath: string, url: string, logger: Logger) {
   return true
 }
 
+const supportedChromiumBrowsers = [
+  'Google Chrome Canary',
+  'Google Chrome Dev',
+  'Google Chrome Beta',
+  'Google Chrome',
+  'Microsoft Edge',
+  'Brave Browser',
+  'Vivaldi',
+  'Chromium'
+]
+
 function startBrowserProcess(browser: string | undefined, url: string) {
   // If we're on OS X, the user hasn't specifically
   // requested a different browser, we can try opening
-  // Chrome with AppleScript. This lets us reuse an
+  // a Chromium browser with AppleScript. This lets us reuse an
   // existing tab when possible instead of creating a new one.
   const shouldTryOpenChromeWithAppleScript =
-    process.platform === 'darwin' && (browser === '' || browser === OSX_CHROME)
+    process.platform === 'darwin' &&
+    (!browser || supportedChromiumBrowsers.includes(browser))
```

Comment:
Maybe we should add a `if (browser === 'google chrome') browser = 'Google Chrome'` before this to keep backwards compat with the previous `OSX_CHROME` option

Reply:
Good catch! I didn't mutate the main variable to not change the behaviour if nothing is opened

Reply:
I would slightly lean to mutating `browser` because it's cleaner and one-less variable, but either way works for me too 😄 Thanks for fixing this!

Reply:
Yeah but it would mean that people using BROWSER='google chrome' on windows will not call the `open` lib with `Google Chrome` and I this could lead to a different behaviour that I can't test